framework/include/base/MooseVariableBase.h

```diff
@@ -85,8 +91,9 @@ class MooseVariableBase
 
   /**
    * Get the order of this variable
+   * Note: Order enum can be implicitly converted to unsigned int.
    */
-  unsigned int order() const;
+  Order order() const;
```

Comment:
Normally I don't like to just change public interfaces in MOOSE like this but it looks like nobody is using this except you  (Yak)?